_docs/bigquery/enabling-billing.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+section: bigquery
+navigation_title: "Enabling billing"
+title: "Enabling billing"
+order: 3
+date: 2020-04-29
+---
+
+<!---
+In this article explained how to Enable BigQuery API if it is not enabled
+-->
+## Before you begin
+* You should have an existing [Google Cloud Platform account ▸](creating-gcp-account)
```

Comment:
Let's use "▸" only if we have call-to-action. Here you just give data with a context.